pkgs/development/python-modules/micawber/default.nix

```diff
@@ -1,19 +1,29 @@
-{ lib, buildPythonPackage, fetchPypi, beautifulsoup4 }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, beautifulsoup4
+, pythonOlder
+}:
 
 buildPythonPackage rec {
   pname = "micawber";
-  version = "0.5.4";
+  version = "0.5.5";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
```

Comment:
why disable when it official supports all versions down to 2.6?

https://pypi.org/project/micawber/

Reply:
Python 3.7 will be EOL two days. The module might work with Python 2.6 but I highly doubt that somebody will confirm it for a Python release that was retired almost a decade ago. Also, upstream only run their CI against supported [Python releases](https://github.com/coleifer/micawber/blob/master/.github/workflows/tests.yaml#L10).